apps/website/src/pages/docs/statusmessage.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Status message
+description: Component to display icon and text below the component.
```

Comment:
what's "the component"?

Reply:
Copied same description as in jsdocs. Jsdocs had combobox as a component which is not true anymore. Not sure what to put here.
"any component"?

Reply:
i think it's useful for form fields, right? for showing error messages or even instructions (like password requirements). so it's worth explaining that.

also could mention the components in which it is used (the ones with a `message` prop)

Reply:
Updated description. Also added a link to this page in InputGrid.